src/main/java/mod/crend/movesubtitles/mixin/SubtitlesHudMixin.java

```diff
@@ -24,20 +24,27 @@ public abstract class SubtitlesHudMixin {
 	@WrapOperation(method = "render", at=@At(value = "INVOKE", target = "Lnet/minecraft/client/util/math/MatrixStack;translate(FFF)V"))
 	private void moveSubtitles(MatrixStack instance, float x, float y, float z, Operation<Void> original, DrawContext context, @Local(ordinal = 0) int i, @Local(ordinal = 1) int j) {
 		int centerX = j / 2;
-		int heightPerColumn = 9;
+		int heightPerRow = 10;
 		float x2 = switch (MoveSubtitlesConfig.INSTANCE.edge) {
 			case TOP_LEFT, LEFT, BOTTOM_LEFT -> centerX + 2.0F;
 			case TOP, BOTTOM -> context.getScaledWindowWidth() / 2.0F;
 			case TOP_RIGHT, RIGHT, BOTTOM_RIGHT -> x;
 		};
 		float y2 = switch (MoveSubtitlesConfig.INSTANCE.edge) {
-			case TOP_LEFT, TOP, TOP_RIGHT -> 15 + i * (heightPerColumn + 1);
-			case LEFT, RIGHT -> context.getScaledWindowHeight() / 2.0F - (this.audibleEntries.size() / 2.0F - i) * (heightPerColumn + 1);
+			case TOP_LEFT, TOP, TOP_RIGHT -> 15 + i * heightPerRow;
+			case LEFT, RIGHT -> (context.getScaledWindowHeight() / 2.0F - heightPerRow / 2.0F - (heightPerRow * i)) + this.audibleEntries.size() * heightPerRow / 2;
```

Comment:
Please double-check my maths: this is equivalent to `case LEFT, RIGHT -> (context.getScaledWindowHeight() / 2.0F + ((this.audibleEntries.size() - 1) / 2.0F - i) * heightPerRow);`, correct? So it'd be a wrong sign + off-by-one error in my original code?

Reply:
Oh yeah, I should've simplified it! I've double checked and your working is correct, will push another commit changing to that.